src/Main/AboutWidget.cpp

```diff
@@ -32,14 +32,18 @@ AboutWidget::AboutWidget(const ProjectInfo &info, QWidget *parent)
     : QDialog{parent} {
   QLabel *label = new QLabel(this);
   QPushButton *close = new QPushButton("Close", this);
-  label->setText(QString("<p><b>%1 %2</b></p>"
-                         "<p>Build on %3</p>"
-                         "<p>From revision <a "
-                         "href=\"%4%5\">%5</a></p>"
-                         "<p>Build type: %6</p>"
-                         "<p>%7</p>")
+  QString text = QString(
+                     "<p><b>%1 %2</b></p>"
+                     "<p>Build on %3</p>"
+                     "<p>From revision <a "
+                     "href=\"%4%5\">%5</a></p>"
+                     "<p>Build type: %6</p>")
                      .arg(info.name, info.version, __DATE__, info.url,
-                          info.git_hash, info.build_type, License()));
+                          info.git_hash, info.build_type);
+  if (info.showLicense) {
+    text += QString("<p>%7</p>").arg(License());
```

Comment:
This looks like a copy paste error. Should %7 be %1?
```suggestion
    text += QString("<p>%1</p>").arg(License());
```